ui-tests/src/test/java/org/openmrs/reference/XSSOnPhoneNumberFieldTest.java

```diff
@@ -36,7 +32,6 @@ public void setup() {
     }
 
     @Test
-    @Ignore
     @Category(BuildTests.class)
     public void XSSOnPhoneNumberFieldTest() throws Exception {
```

Comment:
Align the `method-name` with the test method naming convention !

Reply:
> Align the `method-name` with the test method naming convention !

Resolved thanks.